File: Miscelaneous/aluno.py
from dataclasses import dataclass


@dataclass
class Aluno:
    nome: str
    matricula: str
    nota_final: float

    def foi_aprovado(self):
        return self.nota_final >= 6.0

    def resumo(self):
        return f"{self.nome} ({self.matricula}): {self.foi_aprovado()}"

File: Miscelaneous/Aluno.java
public class Aluno {
    private final String nome;
    private final String matricula;
    private final double notaFinal;

    public Aluno(String nome, String matricula, double notaFinal) {
        this.nome = nome;
        this.matricula = matricula;
        this.notaFinal = notaFinal;
    }

    public boolean foiAprovado() { return notaFinal >= 6.0; }
    public String resumo() { return nome + " (" + matricula + "): " + foiAprovado(); }
}